.npmignore

```diff
@@ -3,27 +3,22 @@
 .editorconfig
 .eslintignore
 .eslintrc.json
-.gitignore
 .github
+.gitignore
 .npmignore
 .nycrc
 .project
 .settings
+.vagrant
```

Comment:
this should no longer exist in the project after https://github.com/fluid-project/infusion/pull/1029